client/store/modules/revisions.js

```diff
@@ -24,6 +26,7 @@ getter(function hasMoreResults() {
 });
 
 action(function fetch() {
+  if (!this.getters) return;
```

Comment:
This feels wrong... Getters are not defined, but the action is already used? Can you explain the use case?

Reply:
This `if` is here to fix render bug in history tab. I think that `getters` are not loaded yet, but still action is triggered. @franosincic suggested this, after this approach, error disappeared.

Reply:
This sounds like it is `vuex-module` related :thinking: However, please put comment above to investigate it further once we migrate to _raw_ `vuex`.